service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionController.java

```diff
@@ -10,24 +10,48 @@
 import org.springframework.web.bind.annotation.RequestHeader;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
 import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
 import uk.gov.hmcts.reform.fpl.events.SubmittedCaseEvent;
+import uk.gov.hmcts.reform.fpl.service.UserDetailsService;
 
 import javax.validation.constraints.NotNull;
+import java.util.Map;
 
 @Api
 @RestController
 @RequestMapping("/callback/case-submission")
 public class CaseSubmissionController {
 
     private final ApplicationEventPublisher applicationEventPublisher;
+    private final UserDetailsService userDetailsService;
 
     @Autowired
-    public CaseSubmissionController(ApplicationEventPublisher applicationEventPublisher) {
+    public CaseSubmissionController(ApplicationEventPublisher applicationEventPublisher, UserDetailsService userDetailsService) {
         this.applicationEventPublisher = applicationEventPublisher;
+        this.userDetailsService = userDetailsService;
     }
 
-    @PostMapping
+    @PostMapping("/about-to-start")
+    public ResponseEntity handleCaseStart(
+        @RequestHeader(value = "authorization") String authorization,
+        @RequestBody CallbackRequest callbackrequest) {
+        CaseDetails caseDetails = callbackrequest.getCaseDetails();
+
+        String label = "I, " + userDetailsService.getUserName(authorization) + ", believe that the facts stated in this application are true.";
+
+        Map<String, Object> data = caseDetails.getData();
+        data.put("submissionConsentLabel", label);
+
+        AboutToStartOrSubmitCallbackResponse body = AboutToStartOrSubmitCallbackResponse.builder()
+            .data(data)
+            .build();
+
+        return ResponseEntity.ok(body);
```

Comment:
Please do: 

```
return AboutToStartOrSubmitCallbackResponse.builder()
             .data(data)
             .build()
```

and change method return type to `AboutToStartOrSubmitCallbackResponse`.